Makefile

```diff
@@ -0,0 +1,35 @@
+# Go parameters
+GOCMD=go
+GOBUILD=$(GOCMD) build
+GOINSTALL=$(GOCMD) install
+GOCLEAN=$(GOCMD) clean
+GOTEST=$(GOCMD) test
+GOGET=$(GOCMD) get
+GOMOD=$(GOCMD) mod
+
+.PHONY: all test coverage
+all: test coverage examples
+
+get:
+	$(GOGET) -t -v ./...
+
+TLS_CERT ?= redis.crt
+TLS_KEY ?= redis.key
+TLS_CACERT ?= ca.crt
+REDISGRAPH_TEST_HOST ?= 127.0.0.1:6379
+
+examples: get
+	@echo " "
+	@echo "Building the examples..."
+	$(GOBUILD) ./examples/redisgraph_tls_client/.
+	./redisgraph_tls_client --tls-cert-file $(TLS_CERT) \
+						 --tls-key-file $(TLS_KEY) \
+						 --tls-ca-cert-file $(TLS_CACERT) \
+						 --host $(REDISGRAPH_TEST_HOST)
```

Comment:
I don't see a reason for a Makefile to run the examples, build sure. run is a bit odd in my mind.

Reply:
solved per new commit